static/docs/commands-reference/diff.md

```diff
@@ -54,10 +90,53 @@ Git reference.
 
 ```dvc
     $ dvc diff HEAD^
-    dvc diff from 6a819f8fa053f124f6a5487efc824a8c17366c71 to 612f6caf5c5daeb172167db285efd8b169d41b60
+    dvc diff from df613bc to ed10968
```

Comment:
@shcheklein please notice I changed this sample output although that change hasn't been implemented in the dvc repo. (Same for line 133.)